client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java

```diff
@@ -818,7 +818,6 @@ public void testSearchAfterWithPointInTime() throws Exception {
 
         ClearScrollResponse closeResponse = client.closePointInTime(new ClosePointInTimeRequest(pitId), RequestOptions.DEFAULT); // <5>
         assertTrue(closeResponse.isSucceeded());
-        // end::search-after-with-point-in-time
```

Comment:
Should we remove also "// <5> " comment on the line 819?

Reply:
I removed this unused doc test.